gems/activerecord/7.0/activerecord-7.0.rbs

```diff
@@ -18,4 +22,26 @@ module ActiveRecord
 
     extend RuntimeRegistry
   end
+
+  module Encryption
+    module EncryptableRecord
+      module ClassMethods
+        @deterministic_encrypted_attributes: untyped
+
+        def encrypts: (*untyped names, ?key_provider: untyped?, ?key: untyped?, ?deterministic: bool, ?support_unencrypted_data: untyped?, ?downcase: bool, ?ignore_case: bool, ?previous: untyped, **untyped context_properties) -> untyped
+
+        def deterministic_encrypted_attributes: () -> untyped
+
+        def source_attribute_from_preserved_attribute: (untyped attribute_name) -> (untyped | nil)
+      end
+
+      def encrypted_attribute?: (untyped attribute_name) -> (false | untyped)
```

Comment:
Since `untyped` includes all of types, simply `untyped` is fine. This is also true for `(untyped | nil)`.